frontend/src/context/WorkspaceContext/WorkspaceContext.tsx

```diff
@@ -23,7 +23,8 @@ export const WorkspaceProvider = ({ children }: Props): JSX.Element => {
 
   // memorize the workspace details for the context
   const value = useMemo<TWorkspaceContext>(() => {
-    const wsId = workspaceId || localStorage.getItem('projectData.id');
+    // const wsId = workspaceId || localStorage.getItem('projectData.id');
+    const wsId = workspaceId || "";
```

Comment:
What's the reasoning for this? This isn't going to cause any issues?

Reply:
<img width="1440" alt="Screenshot 2023-04-28 at 7 47 14 PM" src="https://user-images.githubusercontent.com/81024263/235168869-64931068-5da0-4faf-a856-803e89bf6a5a.png">

The reason behind this is, even if the user went to the organization with no projects, he is still going to see the sidebar which have secrets, members, etc. which doesn't make sense right? since you don't have any projects yet in that org.

previously this is causing because their is always a workspaceId in localstorage and the condition that renders the sidebar inside `AppLayout.tsx` i.e. `currentWorkspace` variable always becomes true and the sidebar will be visible even if there is no projects in the organization. 

what i did is simple remove getting of `wsId` from local storage which resolve this issue. 

let me know if you understand it clearly !! @dangtony98